packages/svelte/src/sdk.ts

```diff
@@ -17,4 +18,45 @@ export function init(options: BrowserOptions): void {
   };
 
   browserInit(options);
+
+  detectAndReportSvelteKit();
+}
+
+/**
+ * Adds a global event processor to detect if the SDK is initialized in a SvelteKit frontend,
+ * in which case we add SvelteKit an event.modules entry to outgoing events.
+ * SvelteKit detection is performed only once, when the event processor is called for the
+ * first time. We cannot perform this check upfront (directly when init is called) because
+ * at this time, the HTML element might not yet be accessible.
+ */
+export function detectAndReportSvelteKit(): void {
+  let detectedSvelteKit: boolean | undefined = undefined;
+
+  const svelteKitProcessor: EventProcessor = event => {
+    if (detectedSvelteKit === undefined) {
+      detectedSvelteKit = isSvelteKitApp();
+    }
+    if (detectedSvelteKit) {
+      event.modules = {
+        svelteKit: '1.0',
```

Comment:
Hmm - feels weird to write `1.0` as that might not be the correct version. Thoughts?

Reply:
I agree, it's weird but we have no way of knowing which SvelteKit version was actually being used (let alone that 1.0 isn't even stable yet). So my train of thought for 1.0 was to simply put there something because afaik we have to supply a version here. But happy to change it to `1.x`, `0.x`, `0` or whatever. Or maybe something like `unknown`/`n/a` ? Though I'm not sure which values are allowed here (for example, how they are indexed).

Reply:
Until we are able to confidently discover correct SvelteKit version I think we can keep it at `1.0`

Reply:
I would maybe leave it as `latest`? Later on we can add in the actual version when we support `SvelteKit`.

Reply:
My worry about leaving it as `1.0` is that we are going to pollute results when we eventually do support `SvelteKit` and start adding the proper version number.

Reply:
fine with me. I was also thinking to convert it to a boolean but I don't want to abuse the module field for that

Reply:
Sure, `latest` is also sounds good to me. As long as the indexer is happy with it.
Our main objective is to find out _if_ SvelteKit is used or not, making the exact version of it lower prio IMO.